macros/src/from_macro.rs

```diff
@@ -629,7 +666,8 @@ pub fn implement_device(item: TokenStream) -> TokenStream {
         .map(|r| device_driver_generation::Buffer {
             name: r.name.to_string(),
             id: r.raw_value,
-            description: r.description.clone(),
+            description: None,
+            attributes: r.attributes.clone(),
```

Comment:
I really do want this description to be filled and have the attributes be purely `cfg`.